contracts/arbitrum/connectors/spell/helpers.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import { ListInterface } from "../../common/interfaces.sol";
+
+contract SpellHelpers {
+	/**
+	 * @dev InstaList
+	 */
+	ListInterface internal constant instaList =
+		ListInterface(0x3565F6057b7fFE36984779A507fC87b31EFb0f09);
```

Comment:
@Richa-iitr I think we can add this in common file?

Reply:
Done.